src/postgres/commands/createPostgresServer/steps/PostgresServerSkuStep.ts

```diff
@@ -0,0 +1,144 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Sku } from "@azure/arm-postgresql/src/models";
+import {
+    AzureWizardPromptStep,
+    IAzureQuickPickItem
+} from "vscode-azureextensionui";
+import { ext } from "../../../../extensionVariables";
+import { localize } from "../../../../utils/localize";
+import { nonNullProp } from "../../../../utils/nonNull";
+import { IPostgresServerWizardContext } from "../IPostgresServerWizardContext";
+
+interface ISkuOption {
+  label: string;
```

Comment:
Can you change the tab width for this file back to 4? After that I think we're good to go 👍

Reply:
haha.. ok, now I know how that happened

Reply:
ok, done 👍🏻